frontend/src/App.tsx

```diff
@@ -47,14 +47,25 @@ export default function App() {
             });
     }
 
+    function deleteQuiz(quizToDelete: Quiz) {
+        axios.delete("/api/quiz/" + quizToDelete.id)
+            .then(function (response) {
```

Comment:
instead of function please try to use lambda functions instead.
```suggestion
.then(response => getAllQuizzes())
```